src/tests/dbus-tests/test_80_filesystem.py

```diff
@@ -879,6 +879,72 @@ def test_protective_part_overwrite_mounted(self):
         part.Unmount(self.no_options, dbus_interface=self.iface_prefix + '.Filesystem')
         disk.Format(self._fs_signature, self.no_options, dbus_interface=self.iface_prefix + '.Block')
 
+    def _test_mountpoint(self, label):
```

Comment:
On second thoughts it's probably better if this function receives the expected mount point as a parameter in addition to the label, I can update the pull request tomorrow.

Reply:
In the end I decided to change it, and I added an additional test for utf-8 labels.

Reply:
> Great, thanks! Good point about exfat and utf-8 labels.

FWIW this works fine in UDisks 2.9.4. This is because in _master_ the formatting is handled by libblockdev which sets `LC_ALL=C`: [here](https://github.com/storaged-project/libblockdev/blob/master/src/utils/exec.c#L215) and [here](https://github.com/storaged-project/libblockdev/blob/master/src/utils/exec.c#L419).

I can open a new issue about this.

Reply:
Reported as https://github.com/storaged-project/libblockdev/issues/846

Reply:
Great, thanks! I was thinking about doing the same.